onboarder/installation/index.rst

```diff
@@ -49,12 +49,37 @@ Build the UI
 
 Clear the Symfony cache and execute the following command to build the UI:
 
+
+If you have access to 'make' commands use :
+.. code-block:: bash
+
+    rm -rf var/cache/*
+    bin/console cache:warmup --env=prod
+    make front
+
+Else,
+
+For >=5.0.0 :
 .. code-block:: bash
 
     rm -rf var/cache/*
     bin/console cache:warmup --env=prod
-    bin/console pim:installer:assets --clean --env=prod && yarn run webpack
+    rm -rf public/bundles public/js
+    bin/console pim:installer:assets --clean
+    rm -rf public/css
+    yarn run less
+    rm -rf public/dist
+    yarn run webpack
+
+
+For < 5.0.0 :
```

Comment:
Just for clarity, maybe switch >=5.0.0 instructions with <5.0.0 block to have it in the right order.